components/header/index.tsx

```diff
@@ -0,0 +1,24 @@
+"use client"
+
+import { usePathname } from "next/navigation"
+import Link from "next/link"
+
+export const Header = () => {
+  const pathname = usePathname()
+  return (
+    <header className="bg-sky-400 text-white font-bold h-8 pl-5">
+      <Link
+        className={`link ${pathname === "/about" ? "underline" : ""}`}
+        href="/about"
+      >
+        Про нас
+      </Link>
+      <Link
+        className={`link ${pathname === "/cards" ? "underline" : ""} ml-5`}
```

Comment:
Instead of `ml-5` for each link you can add className to the header `flex items-center gap-5`. So you will have space between elements and you will not need margin anymore.